generators/client-2/templates/src/main/webapp/app/components/util/_truncate-characters.filter.ts

```diff
@@ -1,37 +1,28 @@
-(function() {
-    'use strict';
-
-    angular
-        .module('<%=angularAppName%>.common')
-        .filter('characters', characters);
-
-    function characters () {
-        return charactersFilter;
-
-        function charactersFilter(input, chars, breakOnWord) {
-            if (isNaN(chars)) {
-                return input;
-            }
-            if (chars <= 0) {
-                return '';
-            }
-            if (input && input.length > chars) {
-                input = input.substring(0, chars);
+export class characters {
```

Comment:
you have to register this as a pipe https://angular.io/docs/ts/latest/guide/pipes.html